README.md

```diff
@@ -22,3 +22,25 @@ let int: u32 = varint
      });
 assert_eq!(300, int);
 ```
+
+### Take Until vs Take While (from Standard Library)
+```rust
+use take_until::TakeUntilExt;
+
+fn main() {
+    let items = [1, 2, 3, 4, -5, -6, -7, -8];
+    let filtered_take_while = items
+        .into_iter()
+        .take_while(|x| *x > 0)
+        .collect::<Vec<i32>>();
+    let filtered_take_until = items
+        .into_iter()
+        .take_until(|x| *x <= 0)
+        .collect::<Vec<i32>>();
+    assert_eq!([1, 2, 3, 4], filtered_take_while.as_slice());
+    assert_eq!([1, 2, 3, 4, -5], filtered_take_until.as_slice());
+}
+```
+
+## MSRV
+The MSRV is `1.56.1` stable.
```

Comment:
Maybe this is something that needs to be added to a test action.

Reply:
The MSRV comes from the 2021 edition. I don't see how you want to test this? Can you please outline your idea?

Reply:
I tested it using `cargo msrv --max-1.56.1` which confirms that version compiles fine.
If newer apis where used since 1.56.1 it might not have been compatible.